src/main/java/org/prebid/server/exception/BlacklistedAccountOrApp.java

```diff
@@ -0,0 +1,15 @@
+package org.prebid.server.exception;
+
+public class BlacklistedAccountOrApp extends RuntimeException {
```

Comment:
i'd like to assume separate exceptions.
Also, it is good practice to suffix with exception name `Exception`.